scripts/irods/database_connect.py

```diff
@@ -186,7 +190,10 @@ def get_database_connection(irods_config):
     os.environ['ODBCSYSINI'] = '/etc'
 
     try:
-        return pypyodbc.connect(connection_string.encode('ascii'), ansi=True)
+        if irods_config.catalog_database_type == 'cockroachdb':
+            return pypyodbc.connect(connection_string.encode('ascii'), ansi=True, autocommit = True)
+        else:
+	    return pypyodbc.connect(connection_string.encode('ascii'), ansi=True)
```

Comment:
one last indentation tweak

Reply:
thanks for catching that. will update commit.